README.md

```diff
@@ -40,7 +41,14 @@ modalForms: Ember.inject.service('modal-forms'),
 isModalActive: Ember.computed.readOnly('modalForms.isModalActive'),
 ```
 
-Change the active state of the modal in your component
+### Modal component
+Inject the [remodal](http://sethbrasile.github.io/ember-remodal/#/styling) and modal-forms services in your component
+```js
+remodal: inject.service(),
+modalForms: inject.service('modal-forms'),
```

Comment:
ember-remodal has built in blur support! 
`Apply the remodal-bg class to anything you want blurred in the background while a modal is open.` It tracks open state on its own! Its 3px blur by default.